payments/swish/direct.md

```diff
@@ -148,8 +148,10 @@ Content-Type: application/json
     },
     "swish": {
         "enableEcomOnly": false,
```

Comment:
This trailing comma needs to go inside the `{% comment %}` because JSON doesn't allow it.